lib/ClangImporter/ClangIncludePaths.cpp

```diff
@@ -171,11 +171,16 @@ createClangArgs(const ASTContext &ctx, clang::driver::Driver &clangDriver) {
   return clangDriverArgs;
 }
 
+static bool shouldInjectGlibcModulemap(const llvm::Triple &triple)
```

Comment:
Can you please also run clang-format on this code?

Reply:
Done. I scoped clang-format to the function inside but not the outside. :)